robottelo/performance/candlepin.py

```diff
@@ -25,9 +24,9 @@ class Candlepin(object):
     """
 
     # parameters for Candlepin Authentication
-    username = conf.properties['foreman.admin.username']
-    password = conf.properties['foreman.admin.password']
-    serverhost = conf.properties['main.server.hostname']
+    username = settings.server.admin_username
```

Comment:
This is evaluating the configuration at import time and is not allowing generating document because if not configured settings.server will be `None`. Still trying to figure out a way to avoid breakage when generating the documentation for the test modules.